src/pages/home/report/ReportActionCompose.js

```diff
@@ -127,6 +128,10 @@ class ReportActionCompose extends React.Component {
                 horizontal: 0,
                 vertical: 0,
             },
+            selection: {
+                start: props.comment.length,
+                end: props.comment.length,
```

Comment:
why is the `start` and `end` the same here?

Reply:
it's the initial cursor position.
if the `start` and `end` not equal that means some text selected from start to end